src/modules/navigator/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ px4_add_module(
 	MAIN navigator
 	STACK_MAIN 1300
 	COMPILE_FLAGS
+		-Wno-address-of-packed-member # TODO: fix in c_library_v2
```

Comment:
Can you double check this one? Navigator shouldn't be touching mavlink at all.

Reply:
Not sure why I thought that one, you're right.